model_compression_toolkit/qunatizers_infrastructure/keras/inferable_quantizers/activation_inferable_quantizers/activation_pot_inferable_quantizer.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2023 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import numpy as np
+
+from model_compression_toolkit.core.common.constants import FOUND_TF
+from model_compression_toolkit.qunatizers_infrastructure.common.base_inferable_quantizer import QuantizationTarget
+
+
+
+if FOUND_TF:
+    import tensorflow as tf
+    from model_compression_toolkit.qunatizers_infrastructure.keras.inferable_quantizers.base_pot_inferable_quantizer \
+        import \
+        BasePOTInferableQuantizer
+    class ActivationPOTInferableQuantizer(BasePOTInferableQuantizer):
```

Comment:
Why unique quantizer for PO2? We implemented it as Symmetric with flag. You can see it look pretty much like symmetric_inferable_quantizer